helm/reana/values.yaml

```diff
@@ -49,6 +49,20 @@ secrets:
   cern:
     sso: {}
   reana: {}
+  keycloak:
+    consumer_key: ""
+    consumer_secret: ""
+
+# Keycloak configuration (also refer to secrets.keycloak.*)
+keycloak_config:
+  enabled: false
+  title: ""
+  desciption: ""
```

Comment:
It is required by the helper atm, but as discussed it would make sense to disregard the helper and set everything manually, then this value would become obsolete.